pkg/appmgmt/interfaces/amprotocol.go

```diff
@@ -81,6 +81,7 @@ type ApplicationMetadata struct {
 	TaskGroups              []v1alpha1.TaskGroup
 	PlaceholderTimeoutInSec int64
 	OwnerReferences         []metav1.OwnerReference
+	SchedulingStyle         string
```

Comment:
change this to the struct `SchedulingParameters` and add `placeholderTimeout` and `gangSchedulingStyle` into the single struct. So it will be easier to extend as well.

Reply:
Taken care